cfgov/jinja2/v1/_includes/organisms/post-preview.html

```diff
@@ -39,13 +39,117 @@
 
    ========================================================================== #}
 
-{% macro render(settings, path) %}
-
 {% import 'macros/share.html' as share with context %}
 {% import 'category-slug.html' as category_slug %}
 {% import 'macros/time.html' as time %}
 {% from 'macros/util/format/url.html' import location_image_url as location_image_url %}
 
+{% if value %}
+    <article class="o-post-preview">
+        <div class="meta-header">
+        <span class="date meta-header_right">
+            Published {{ time.render(value.published_date, {'date':true}) }}
+        </span>
+            {% if value.post_category_1 %}
```

Comment:
I wasn't sure about small screens, but looking at the spec it seems the date does come first so you're all set.